package.json

```diff
@@ -10,7 +10,9 @@
   },
   "scripts": {
     "build": "gulp",
-    "test": "jest"
+    "test": "jest",
+    "start": "webpack --mode development",
```

Comment:
seems more like `build` task

Reply:
```

Reply:
Thanks for your comment, I'lI add those changes. Previously I was thinking about webpack-dev-server and I didn't add it because gulp doesn't do it in this repo.